Code/Editor/EditorViewportWidget.cpp

```diff
@@ -102,6 +102,21 @@
 AZ_CVAR(
     bool, ed_visibility_logTiming, false, nullptr, AZ::ConsoleFunctorFlags::Null, "Output the timing of the new IVisibilitySystem query");
 
+namespace
+{
+    void CVar_OnDefaultNearFarChange([[maybe_unused]]const float& value)
+    {
+        EditorViewportWidget* primaryViewport = EditorViewportWidget::GetPrimaryViewport();
+        if (primaryViewport != nullptr)
+        {
+            primaryViewport->OnDefaultCameraNearFarChange();
+        }
+    }
+}
+
+AZ_CVAR(float, ed_editor_camera_near, 0.1f, CVar_OnDefaultNearFarChange, AZ::ConsoleFunctorFlags::Null, "Set editor camera near plane distance");
+AZ_CVAR(float, ed_editor_camera_far, 100.f, CVar_OnDefaultNearFarChange, AZ::ConsoleFunctorFlags::Null, "Set editor camera far plane distance");
```

Comment:
Instead of adding an AZ_CVAR (more of a debug thing) we could add this to the Settings Registry. If you go to EditorViewportSettings.h/cpp, you can see examples like...

```
constexpr AZStd::string_view CameraRotateSmoothingSetting = "/Amazon/Preferences/Editor/Camera/RotateSmoothing";
```

So we could have...

```
constexpr AZStd::string_view CameraNearPlaneDistanceSetting = "/Amazon/Preferences/Editor/Camera/NearPlaneDistance";
constexpr AZStd::string_view CameraFarPlaneDistanceSetting = "/Amazon/Preferences/Editor/Camera/NearPlaneDistance";
```

And then we just write simple getters/setters from there and can call them when we need them. We can also use an AZ::Event like we're doing with `EditorViewportSettingsCallbacks` in `EditorViewportSettings.h` for the callback functionality IIRC. 

Have a look and see what you think, it would just be super handy to do it this way as we could then expose it to the Editor UI in future much more easily (see the Camera settings panel in Global Preferences for an example).